code/ui/.storybook/main.ts

```diff
@@ -32,6 +33,10 @@ const config: StorybookConfig = {
   core: {
     disableTelemetry: true,
   },
+  viteFinal: (vite) => ({
+    ...vite,
+    plugins: [...(vite.plugins || []), csfPlugin({})],
```

Comment:
@shilman coming back to this.
Why are we explicitly adding this to our UI SB instead of adding it per default through `builder-vite` ?